static/src/deploys-radiator/app/render.ts

```diff
@@ -0,0 +1,173 @@
+        const shouldShowProjectNames = !groupDeploys.equals(currentDeploys);
+
+        return h(
+            'li',
+            { className: `deploy deploy--${deployGroup.status.split(' ').join('-').toLowerCase()}` },
+            [
+                h('h2', [
+                    h('a', { href: createBuildLink(deployGroup.build) }, `${deployGroup.build}`)
+                ]),
+                // Only show project names if we have multiple deployed groups
+                exp(shouldShowProjectNames) && ih('ul', {}, groupDeploys
+                    .sortBy(build => build.projectName)
+                    .map(deploy => {
+                        const previousBuild = previousDeploysMap.get(deploy);
+                        return h('li', [
+                            h('a', {
+                                href: createRiffRaffDeployLink(deploy.uuid),
+                                title: previousBuild ? `Previous build: ${previousBuild.build}` : ''
+                            }, deploy.projectName)
+                        ]);
+                    })
+                    .toList()
+                )
+            ]
+        );
+    };
+
+    const createDeployGroup = (deploys: List<DeployRecord>) => (
+        deploys.groupBy(deploy => (
+            createDeployGroupRecord({
+                status: deploy.status,
+                build: deploy.build
+            })
+        ))
+    );
+
+    const currentDeployGroupNodes = createDeployGroup(currentDeploys)
+        .map(renderGroupDeployNodes)
+        .toList();
+
+    return h('div', {}, [
+        ih('ul', { className: 'deploys' }, currentDeployGroupNodes),
+        exp(notRunningDeploys.size > 0) && [
+            h('h2', 'Queue'),
+            ih('ul', {}, createDeployGroup(notRunningDeploys)
+                .map((groupDeploys, deployGroup) => (
+                    h('li', [
+                        h('strong', [
+                            h('a', { href: createBuildLink(deployGroup.build) }, deployGroup.build.toString())
+                        ]),
+                        ih('ul', {}, (
+                            groupDeploys
+                                .sortBy(build => build.projectName)
+                                .map(deploy => {
+                                    const previousBuild = previousDeploysMap.get(deploy);
+                                    return h('li', [
+                                        h('a', {
+                                            href: createRiffRaffDeployLink(deploy.uuid),
+                                            title: previousBuild ? `Previous build: ${previousBuild.build}` : ''
+
+                                        }, deploy.projectName)
+                                    ]);
+                                })
+                                .toList()
+                        ))
+                    ])
+                ))
+                .toList()
+            )
+        ]
+    ]);
+};
```

Comment:
Thanks for the feedback, I agree! I will create a model and use that in another PR.